packages/blaze/builtins.js

```diff
@@ -30,6 +30,41 @@ Blaze.With = function (data, contentFunc) {
 };
 
 /**
+ * Attaches bindings to the instantiated view.
+ * @param {Object} bindings A dictionary of bindings, each binding name
+ * corresponds to a value or a function that will be reactively re-run.
+ * @param {View} view The target.
+ */
+Blaze._attachBindingsToView = function (bindings, view) {
+  view.onViewCreated(function () {
+    _.each(bindings, function (binding, name) {
+      view._scopeBindings[name] = new ReactiveVar;
+      if (typeof binding === 'function') {
+        view.autorun(function () {
+          view._scopeBindings[name].set(binding());
+        }, view.parentView);
+      } else {
+        view._scopeBindings[name].set(binding);
+      }
+    });
+  });
+};
+
+/**
+ * @summary Constructs a View setting the local lexical scope in the block.
+ * @param {Function} bindingsFunc A function to reactively re-run. The returned
+ * dictionary maps names of bindings with values or computations to reactively
+ * re-run.
+ * @param {Function} contentFunc A Function that returns [*renderable content*](#renderable_content).
+ */
+Blaze.Let = function (bindingsFunc, contentFunc) {
+  var view = Blaze.View('let', contentFunc);
+  Blaze._attachBindingsToView(bindingsFunc(), view);
```

Comment:
I have to look into what's going on here more, but we shouldn't be calling a function like `bindingsFunc()` from this function (a View factory) if it's a function that is to be reactively re-rendered.